src/WebJobs.Script.WebHost/Models/HttpOperationInfo.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.Collections.ObjectModel;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.Azure.WebJobs.Script.WebHost.Models
+{
+    [JsonObject]
+    public class HttpOperationInfo
+    {
+        [JsonProperty(PropertyName = "operationId")]
+        public string OperationId { get; set; }
+
+        [JsonProperty(PropertyName = "produces")]
+        public Collection<string> Produces { get; } = new Collection<string> { };
+
+        [JsonProperty(PropertyName = "consumes")]
+        public Collection<string> Consumes { get; } = new Collection<string> { };
+
+        [JsonProperty(PropertyName = "parameters")]
+        public Collection<HttpOperationInputParameterInfo> InputParameters { get; set; }
```

Comment:
There won't be output parameters, right? So we could rename HttpOperationInputParameterInfo to HttpOperationParameterInfo